src/transformers/models/deberta_v2/modeling_deberta_v2.py

```diff
@@ -561,12 +565,12 @@ def build_relative_position(query_size, key_size, bucket_size=-1, max_position=-
         `torch.LongTensor`: A tensor with shape [1, query_size, key_size]
 
     """
-    q_ids = np.arange(0, query_size)
-    k_ids = np.arange(0, key_size)
-    rel_pos_ids = q_ids[:, None] - np.tile(k_ids, (q_ids.shape[0], 1))
+    q_ids = torch.arange(0, query_size)
+    k_ids = torch.arange(0, key_size)
+    rel_pos_ids = q_ids[:, None] - k_ids.repeat(q_ids.shape[0], 1)
```

Comment:
Can't we use broadcast here?

Reply:
Sure, implemented broadcasting in new commit